api/capi/include/nntrainer.h

```diff
@@ -75,6 +75,22 @@ typedef enum {
   ML_LAYER_TYPE_UNKNOWN    /**< Unknown Lyaer */
 } ml_layer_type_e;
 
+/**
+ * @brief Enumeration for the neural network layer property
+ *        InputLayer has 0, 1, 2, 3 properties.
+ *        FullyConnectedLayer has 0, 1, 4 properties.
+ *        BatchNormalizationLayer has 0, 1, 5 properties.
+ * @since_tizen 6.x
+ */
+typedef enum {
+  input_shape = 0,     /**< Input Shaep */
+  bias_zero = 1,       /**< Fill Bias with Zero  */
+  normalization = 2,   /**< Noramlization Flag  */
+  standardization = 3, /**< Standardization Flag  */
+  activation = 4,      /**< Actication for Layer  */
+  epsilon = 5          /**< Epsilon for Batch Normalization  */
```

Comment:
Q. Is there any reason this is not upper case?

Reply:
which one is better ml_nnlayer_set_property(input_size, "1:2:3:4") or ml_nnlayer_set_property(INPUT_SIZE, "1:2:3:4"). 
I think first one is better.

Reply:
Ah.. that sounds reasonable! I prefer the first one too :)
Since I am not an expert on C/C++, would this be safe from namespace polluting in C? 
e.g, if someone is trying to declare `int input_size`?

Reply:
fixed. Thanks.